modules/serverless-installing-cli-web-console.adoc

```diff
@@ -42,3 +44,32 @@ $ tar -xf <file>
 ----
 $ echo $PATH
 ----
+
+.Verification
+
+* Run the following commands to check that the correct Knative CLI resources and route have been created:
++
+[source,terminal]
+----
+$ oc get ConsoleCLIDownload
+----
++
+.Example output
+[source,terminal]
+----
+NAME                  DISPLAY NAME                                             AGE
+kn                    kn - OpenShift Serverless Command Line Interface (CLI)   2022-09-20T08:41:18Z
+oc-cli-downloads      oc - OpenShift Command Line Interface (CLI)              2022-09-20T08:00:20Z
+----
++
+[source,terminal]
+----
+$ oc get route -n openshift-serverless
+----
++
+.Example output
+[source,terminal]
+----
+NAME   HOST/PORT                                                                            PATH   SERVICES                      PORT       TERMINATION     WILDCARD
+kn     kn-openshift-serverless.apps.ci-ln-0r8c2st-72292.origin-ci-int-gce.dev.rhcloud.com          knative-openshift-metrics-3   http-cli   edge/Redirect   None
```

Comment:
Should we use a more abstract example hostname, like, e.g. `kn-openshift-serverless.apps.example.com`?  

We don't want to expose details of internal infrastructu

Reply:
SGTM, can you review any examples here and suggest abstract / example values for them please?